lib/grpc/server/adapters/cowboy/handler.ex

```diff
@@ -449,8 +449,8 @@ defmodule GRPC.Server.Adapters.Cowboy.Handler do
     end
   end
 
-  def info({:EXIT, pid, :normal}, req, state = %{pid: pid}) do
-    exit_handler(pid, :normal)
```

Comment:
removed this call to `exit_handler` -- cause this function is redundant for this callback.